CONTRIBUTOR.md

```diff
@@ -0,0 +1,3 @@
+# Contributing
+
+We welcome contributions from everyone. More details on how to contribute will be coming. For now simply submit a pull-request or create an issue.
```

Comment:
@Diogenesoftoronto we can't actual accept contributions without a signed contributors agreement (See https://github.com/artefactual/archivematica/blob/stable/1.14.x/CONTRIBUTING.md#contributors-agreement for an example).  I think it's better to omit the contributor file for now, and add one later when we have worked out the contents.

Reply:
Sensible